.github/workflows/integration.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - run: npm ci
       - run: sudo apt-get install xvfb
       - run: npm run dev &
-      - uses: ifaxity/wait-on-action@v1
+      - uses: BIZFactoryGmbH/wait-on-action@v1
```

Comment:
Hmm. Still using the `v1` seems a bit odd. Maybe change that to `v2`? Or would that be inappropriate? :thinking:
Did you "just" make it work again or did you also update the dependencies?

Reply:
Talking about the fork - I like the idea: ifaxity released a new version (also comes with dependency bumps) just yesterday: https://github.com/iFaxity/wait-on-action/releases/tag/v1.1.0

Reply:
```suggestion
      - uses: ifaxity/wait-on-action@v1.1.0